src/openms/include/OpenMS/ANALYSIS/ID/FragmentIndexScorer.h

```diff
@@ -150,7 +149,7 @@ namespace OpenMS
     /** @brief places the k-largest elements in the front of the input array. Inside of the k-largest elements and outside the elements are not sorted
      * Brazenly stolen from sage.
```

Comment:
I would remove this "stolen" but at the same time more clearly state that the code is inspired and partially directly translated from the rust Sage search engine by Michael R. Lazear.